npm/ng-packs/packages/core/src/lib/tests/permission.guard.spec.ts

```diff
@@ -18,6 +29,10 @@ describe('PermissionGuard', () => {
   let httpErrorReporter: SpyObject<HttpErrorReporterService>;
   let permissionService: SpyObject<PermissionService>;
 
+  const mockOAuthService = {
+    isAuthenticated: true,
+  };
```

Comment:
Is that a true mocking method ? Can we provide another way for mocking a service ? It seems like duplicated in DummyComponent 🤔 ?